### ISSUE TITLE
Linter is showing errors that don't exist after update

### ISSUE THREAD

**Issue by IngCr3at1on** [joefitzgerald/gometalinter-linter#52]
I updated Atom and subsequently all my installed packages today and now I'm getting an exuberant amount of false errors reported on my Go files.

Atom loads properly without any errors and I can even confirm `go get` works because it had to download dlv. Right now I'm looking at a file that is reporting 323 issues pretty much all of which are related to gometalinter not being able to find imported packages; including but not limited to core packages:

```
gometalinter
Error
could not import bytes (reading export data: /usr/lib/go/pkg/linux_amd64/bytes.a: unknown version: v1    bytes
5$GOROOT/src/bytes/buffer.go�Buffer) (gotype)at line 4 col 2
gometalinter
Error
could not import encoding/json (reading export data: /usr/lib/go/pkg/linux_amd64/encoding/json.a: unknown version: v1json  E$GOROOT/src/encoding/json/decode.go�Un) (gotype)at line 5 col 2
gometalinter
Error
could not import fmt (reading export data: /usr/lib/go/pkg/linux_amd64/fmt.a: unknown version: v1fmt
-$GOROOT/src/fmt/scan.go�  State pr) (gotype)at line 6 col 2
gometalinter
Error
could not import io/ioutil (reading export data: /usr/lib/go/pkg/linux_amd64/io/ioutil.a: unknown version: v1ioutil    =$GOROOT/src/io/ioutil/ioutil.goR
ReadA) (gotype)at line 7 col 2
gometalinter
Error
could not import net/http (reading export data: /usr/lib/go/pkg/linux_amd64/net/http.a: unknown version: v1http
A$GOROOT/src/net/http/transport.go�!Cli) (gotype)at line 8 col 2
gometalinter
Error
could not import net/url (reading export data: /usr/lib/go/pkg/linux_amd64/net/url.a: unknown version: v1url
3$GOROOT/src/net/url/url.go�   Error�) (gotype)at line 9 col 2
gometalinter
Error
could not import strconv (reading export data: /usr/lib/go/pkg/linux_amd64/strconv.a: unknown version: v1
strconv    5$GOROOT/src/strconv/atob.goParseBoo) (gotype)at line 10 col 2
gometalinter
Error
could not import strings (reading export data: /usr/lib/go/pkg/linux_amd64/strings.a: unknown version: v1
strings    ;$GOROOT/src/strings/compare.go
Compa) (gotype)at line 11 col 2
gometalinter
Error
could not import time (reading export data: /usr/lib/go/pkg/linux_amd64/time.a: unknown version: v1time3$GOROOT/src/time/format.got ANSIC2-/Mon) (gotype)at line 12 col 2
```

```
atom --version                                                                                                                                                                                               
Atom    : 1.9.9
Electron: 0.37.8
Chrome  : 49.0.2623.75
Node    : 5.10.0
```

```
Linux <host> 4.7.1-1-ARCH #1 SMP PREEMPT Wed Aug 17 08:13:35 CEST 2016 x86_64 GNU/Linux
```

**Comment by joefitzgerald** [comment]
This looks really weird. gometalinter has been vendoring its linters, as of a recent version. Perhaps try `go get -u github.com/alecthomas/gometalinter && gometalinter --install`?

**Comment by joefitzgerald** [comment]
Also, I would _highly recommend disabling use of gotype_.

**Comment by IngCr3at1on** [comment]
@joefitzgerald running that command reduced my overall project issues from 1446 to 717 so that's progress...

Silly question, how to go about disabling gotype? lol

**Comment by joefitzgerald** [comment]
@IngCr3at1on:  https://github.com/joefitzgerald/gometalinter-linter#configuration

**Comment by IngCr3at1on** [comment]
@joefitzgerald thanks a ton that did it.
